service/src/main/java/bio/terra/policy/service/pao/graph/model/ExplainGraphNode.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.policy.service.pao.graph.model;
+
+import bio.terra.policy.common.model.PolicyInput;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/** The ExplainGraphNode represents the contributions of one object to a policy. */
```

Comment:
The objectId is the UUID handed in when the PAO is created. So for a workspace, it is the workspaceUuid. Since that is the only object we have, right now they are all workspaceUuid.

It is also the key to looking up a PAO, so I also call it a PaoId. Maybe we should setting on one name and update the code to be consistent. I think it makes sense that it is objectId in the REST API.